tasks/python3/getVisMetadata.py

```diff
@@ -48,6 +65,8 @@ def main(url):
     except Exception as e:
       print("%s:" % (e))
 
+  print(layer_metadata)
```

Comment:
@jasontk19 I feel like we may want to remove some of these prints. I am getting a printout of a massive object and I think this could make it harder to debug other issues.

Reply:
Whoops yeah that shouldn't still be in there, good catch